### DISCUSSION TITLE
How to customize log params in mlflow.

### DISCUSSION THREAD

**Discussion by itomoko77** [pycaret/pycaret#2460] [Q&A]
Using log_experiment = True and experiment_name = 'xxxx' in setup, we can integrate with mlflow and all logs are shown up in mlflow. By default, run name is models' name. Is it possible to customize Run name using like mlflow.start_run(run_name=xxxx)?
Also we would like to log train data in csv (or zip) and train result in text to artifact. If log_data = True in setup, is it possible to do that?

**Comment by kretes** [comment]
I'm missing the option of `customize Run name` as well
